src/utils/dispute-utils.js

```diff
@@ -208,26 +225,36 @@ export function getPhaseAndTransition(dispute, courtConfig, nowDate) {
   }
 }
 
+/**
+ * Tells the adjudication state of a round at a certain time.
+ * @param {Object} dispute Dispute querying the adjudication round of
+ * @param {Object} round The round that is being queried
+ * @param {Number} now Current time in ms
```

Comment:
It seems weird that a previous API has the `now` be a Date object whereas here (and in some other functions) it's a number. We should pick one and stick with it across the internal API.

Reply:
Yep totally makes sense.